spec/nylas/webhook_spec.rb

```diff
@@ -0,0 +1,199 @@
+# frozen_string_literal: true
```

Comment:
If this is a test file, can you name it as `webhook_spec_test.rb`?

Reply:
That's a good point, this was just the file structure of the SDK when I took over. I'll note it down for a future update, I just don't want to make such changes in this PR. Thanks for that feedback!